core/aws_ddk_core/stages/sqs_lambda.py

```diff
@@ -36,8 +36,8 @@ def __init__(
         scope: Construct,
         id: str,
         environment_id: str,
-        code: Code,
-        handler: str,
+        code: Optional[Code] = None,
```

Comment:
Let me know if there is a better way to handle conditionally optional arguments.

Reply:
We could have a method that validates and handles these arguments but I feel it's an overkill for now